lib/logflare_web/endpoint.ex

```diff
@@ -1,15 +1,20 @@
 defmodule LogflareWeb.Endpoint do
   use Phoenix.Endpoint, otp_app: :logflare
+
   @session_options [store: :cookie, key: "_logflare_key", signing_salt: "INPMyhPE"]
 
   socket("/socket", LogflareWeb.UserSocket, websocket: [compress: true])
 
   socket("/logs", LogflareWeb.LogSocket, websocket: [compress: true])
 
-  socket("/live", Logflare.LiveView.Socket,
-    websocket: [connect_info: [session: @session_options], compress: true]
+  socket("/live", Phoenix.LiveView.Socket,
```

Comment:
removal of Logflare.LiveView.Socket seems separate to the playwright addition, can we move it to a separate PR please